src/templates/offering.js

```diff
@@ -30,7 +30,7 @@ const OfferingTemplate = ({ data }) => {
     Modal.setAppElement('#___gatsby')
 
     const offering = data.allStrapiOffering.nodes[0];
-
+    console.log(offering);
```

Comment:
Needs debug statement cleanup. Can happen in subsequent tickets before the next release.